sssd_test_framework/misc/__init__.py

```diff
@@ -97,3 +98,16 @@ def to_list_without_none(r_list: list[Any]) -> list[Any]:
     :rtype: list[Any]
     """
     return [x for x in r_list if x is not None]
+
+
+def parse_ldb_output(output: str) -> list[list[str]]:
```

Comment:
ldb prints the output in ldif format so `parse_ldif` would be a better name. Also please add unit tests for this.

Reply:
And it should return the records in dict format directly, ie. instead of calling `attrs_parse(parse_ldb_output(...))` you want to get the result directly from `parse_ldb_output(...)`.